code/modules/clothing/under/miscellaneous.dm

```diff
@@ -751,11 +751,12 @@
 	item_state = "noble_clothes"
 
 /obj/item/clothing/under/contortionist
-	name = "Contortionist's Jumpsuit"
+	name = "atmospheric technician's jumpsuit"

```

Comment:
Presumably should be capitalised (i.e. Atmospheric Technician's Jumpsuit)?

Reply:
nope, that would be non-standard and different from the non-traitor suit.